docs/config/config-files.md

```diff
@@ -29,6 +29,12 @@ The list of the words that are being used to generate [CAPTCHA](security/captcha
 
 This file contains an array of foreign characters for transliteration conversion used by the Text helper (example would be generating URL Titles for entries).
 
+#### HTML Form Attributes
+
+`valid_form_attributes.php`
```

Comment:
@intoeetive Should this be a .php file reference?  Or more of a embed?

Reply:
@TomJaeger it's name of config file. No links of embeds here